frontend/herd_view.tsx

```diff
@@ -28,6 +38,14 @@ export function HerdView({id}: {id: string}) {
     {herd &&
       <>
         <h2>{`G${herd.herd}`} {herd.herd_name ? `- ${herd.herd_name}` : ''}</h2>
+        <dl>
+          {fields.map(field => <>
```

Comment:
It's only after `return` you need `<> ... </>` or `( ... )`, not when you're in an unfinished expression such as to the right of `=>`.

Reply:
Oh I didn't see you had both `<dt>` and `<dd>`, then you need a `<>`-fragment of course.

Reply:
yeah, but I had to change it to a div to get a key on it, so I still did it wrong ^^